docs/admin/reference/backup.rst

```diff
@@ -152,12 +152,16 @@ measures it requires before any additional VMs are configured.
   the VM.** Continue through the reinstallation process. The correct template will be
   configured as you follow the rest of these instructions.
 
-  If you are restoring your own customized VMs and templates, you may need to take
-  additional steps, such as ensuring your templates are supported. Follow the Qubes
-  documentation on upgrading templates (for example:
-  `Fedora templates <https://www.qubes-os.org/doc/templates/fedora/#upgrading>`_,
+  If you are restoring your own customized VMs and templates, you will need to take
+  additional steps. You may decide to create new templates for your custom VMs and
+  provision them with the necessary applications/customizations (recommended), or
+  you may upgrade your existing templates following the upstream documentation
+  (`Fedora templates <https://www.qubes-os.org/doc/templates/fedora/#upgrading>`_,
   `Debian templates <https://www.qubes-os.org/doc/templates/debian/#upgrading>`_),
-  or contact Support.
+  then upgrade their package repositories to the Qubes 4.2 repoistories using
+  ``qubes-dist-upgrade --template-standalone--upgrade`` (more information can be
```

Comment:
It looks like this tool has to be installed first with `sudo qubes-dom0-update -y qubes-dist-upgrade` -- on my system it was not present, and the linked docs instruct the user to install it, but we should probably say so here as well.

Typo: `--template-standalone--upgrade` (double hyphen)